grammar.js

```diff
@@ -1010,19 +1014,21 @@ module.exports = grammar({
     _list_destructing: $ => seq(
       'list',
       '(',
-      commaSep(optional(choice(
+      commaSep1(optional(choice(
         choice(alias($._list_destructing, $.list_literal), $._variable),
         seq($._expression, '=>', choice(alias($._list_destructing, $.list_literal), $._variable))
       ))),
+      optional(','),
```

Comment:
It seems like this `optional(',')` rule is adding ambiguity.

Do you need this, since the values inside the `commaSep1` are optional? I would think that a trailing comma would match the `commaSep1`.

Reply:
You're perfectly right! Thanks for stewarding this project and sharing your knowledge!